[Yaml] Add the `Yaml::DUMP_FORCE_DOUBLE_QUOTES_ON_VALUES` flag to enforce double quotes around string values (#59880)
symfony/symfony: src/Symfony/Component/Yaml/CHANGELOG.md

---

 * Add compact nested mapping support by using the `Yaml::DUMP_COMPACT_NESTED_MAPPING` flag

>  * Add force dumping with quotes support by using the `Yaml::DUMP_FORCE_QUOTES` flag

Reviewer: I would rephrase it like this:

```suggestion
 * Add always-quoted string dumping support by using the `Yaml::DUMP_FORCE_QUOTES` flag
```

Author: Good notes, thanks!

Reviewer 2: I would even reword this a bit more:

```suggestion
 * Add the `Yaml::DUMP_FORCE_QUOTES` flag to enforce double quotes around strings
```
